_pages/how-we-work/tools/google-calendar.md

```diff
@@ -14,38 +14,24 @@ TTS uses **shared calendars** for events, interviews, and "out of office" time.
 
 The following calendars are used for internal communication:
 
-* [TTS Classes](https://calendar.google.com/calendar/embed?src=gsa.gov_gi2e1fc66fh4v5in8ogsm0v8oo%40group.calendar.google.com): New hire and skill-bases training sessions.
-* [TTS Guilds and Working Groups](https://www.google.com/calendar/embed?src=gsa.gov_o1aqcv28k1f0nmca5bkch8los4%40group.calendar.google.com): Meetings and other events pertaining to working groups and guilds.
+- [TTS Classes](https://calendar.google.com/calendar/embed?src=gsa.gov_gi2e1fc66fh4v5in8ogsm0v8oo%40group.calendar.google.com): New hire and skill-bases training sessions.
+- [TTS Guilds and Working Groups](https://www.google.com/calendar/embed?src=gsa.gov_o1aqcv28k1f0nmca5bkch8los4%40group.calendar.google.com): Meetings and other events pertaining to working groups and guilds.
 
 **18F also uses these shared calendars:**
 
-* [18F Events](https://calendar.google.com/calendar/embed?src=gsa.gov_3rapmucstu32ma18da84el20ug%40group.calendar.google.com): Team-wide events, start dates, standups, and so on.
-* 18F Interviews: New hire interviews.
-* [18F Out of office](https://calendar.google.com/calendar/embed?src=gsa.gov_bth7useo0eeiicjgos2di6ph8k%40group.calendar.google.com): In addition to <em>officially</em> requesting time off in <a href="/gsa-internal-tools/#hr-links">HR Links</a>, please also add an event on the Google calendar "18F Out of Office" and invite yourself, so we all know when you’re gone.
+- [18F Events](https://calendar.google.com/calendar/embed?src=gsa.gov_3rapmucstu32ma18da84el20ug%40group.calendar.google.com): Team-wide events, start dates, standups, and so on.
+- 18F Interviews: New hire interviews.
+- [18F Out of office](https://calendar.google.com/calendar/embed?src=gsa.gov_bth7useo0eeiicjgos2di6ph8k%40group.calendar.google.com): In addition to <em>officially</em> requesting time off in <a href="/gsa-internal-tools/#hr-links">HR Links</a>, please also add an event on the Google calendar "18F Out of Office" and invite yourself, so we all know when you’re gone.
```

Comment:
I'm surprised the `gsa-internal-tools` link is resolving correctly and not throwing errors without baseurl

Reply:
Good catch! I'll fix this separately.

Reply:
https://github.com/18F/handbook/pull/1465

Reply:
https://github.com/18F/handbook/pull/1465